docs/cristin-proxy-swagger.yaml

```diff
@@ -666,6 +666,8 @@ paths:
       summary: Find person by identifier
       description: Returns a single person matching id parameter
       operationId: GetPersonById
+#      security:
```

Comment:
?

Reply:
Endpoint is currently  without authorization and without NIN in response